package.json

```diff
@@ -22,6 +22,7 @@
     "typescript": "^4.9.3"
   },
   "dependencies": {
+    "kafkajs": "^2.2.3",
```

Comment:
isn't `kafkajs` already included?

Reply:
I think it is there because this PR was before your merge and I had to include it

Reply:
Can we resolve this conversation @Frawless ?

Reply:
it is already in `devDependencies`, why it Is needed here?

Reply:
ok I remove it from `package.json`

Reply:
it is still in lock file

Reply:
@mmusil  it is something what should be removed as well? I mean the change for the file

Reply:
removed from `package-lock.json`

Reply:
Can we resolve this conversation @Frawless ?